fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/PushDownTopNBelowOuterJoinRule.java

```diff
@@ -102,8 +105,7 @@ public List<OptExpression> transform(OptExpression input, OptimizerContext conte
                 .setOrderByElements(topn.getOrderByElements())
                 .setLimit(topn.getLimit())
                 .setTopNType(topn.getTopNType())
-                .setSortPhase(topn.getSortPhase())
-                .setIsSplit(false)
```

Comment:
why remove spilt false flag?

Reply:
`false ` is the default value. no need add it.